core/src/DevStep.cpp

```diff
@@ -12,6 +12,8 @@
 
 namespace Nextsim {
 
+const Duration DevStep::myriadJulianYears = Duration(3.15576e11);
```

Comment:
This seems  a bit arbitrary solution to me, even though it will work. Can we consider one of two alternatives?

1. Change line 39 of the modified DevStep.cpp to read ` if ((m_restartPeriod > 0) || (mData->time() >= lastOutput + m_restartPeriod)) {` and then a RESTARTPERIOD_KEY with a value 0 gives no restart files (and use this as default).
2. Change line 39 of the modified DevStep.cpp to read ` if ((m_restartPeriod >= 0) || (mData->time() >= lastOutput + m_restartPeriod)) {` and then a RESTARTPERIOD_KEY with a negative value gives no restart files (and use this as default).

Reply:
Choice 1, except with AND `&&` rather than OR `||`